src/gsy_e/models/strategy/external_strategies/pv.py

```diff
@@ -315,6 +315,13 @@ def _list_offers_aggregator(self, arguments: Dict) -> Dict:
         return response
 
     def _offer_aggregator(self, arguments: Dict) -> Dict:
+        response_message = ""
+        arguments, filtered_fields = self.filter_degrees_of_freedom_arguments(arguments)
+        if filtered_fields:
+            response_message = (
+                "The following arguments are not supported for this market and have been "
+                f"removed from your order: {filtered_fields}.")
+
```

Comment:
This looks to me that it could also be put into a decorator and consequently used more easily in multiple places. What do you think?

Reply:
We've briefly talked about it [here](https://github.com/gridsingularity/gsy-e/pull/1269#discussion_r738560687). @Hassan754 suggested to leave it as it is for now, since new changes will come soon.